src/System.Buffers/src/CompatibilitySuppressions.xml

```diff
@@ -0,0 +1,12 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- https://learn.microsoft.com/en-us/dotnet/fundamentals/package-validation/diagnostic-ids -->
+<Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
```

Comment:
Is this the fix for the APICompat failures I was seeing in the maestro deps flow PR? https://github.com/dotnet/maintenance-packages/pull/17/checks?check_run_id=18283464749

Or is it fixing something else?

Reply:
This file contains the suppressions for the TFM breaking changes. I think this is different to your build errors.